test/integration/config/esm-bad/index.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
+
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* eslint-disable */
+
+import newrelic from '../../../../index.js'
```

Comment:
after fixing all the linting errors with the comment above it's still failing here because you're importing something and not using it

Reply:
Okay, made a change to actually use it.